servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -213,17 +218,71 @@ public int deleteExpiredX509CertRecords(int expiryTimeMins) {
     public boolean updateUnrefreshedCertificatesNotificationTimestamp(String lastNotifiedServer,
                                                                       long lastNotifiedTime,
                                                                       String provider) {
-        // Currently unimplemented for AWS
-        return false;
+        try {
+            ScanResult result = getRecordsWithUnrefreshedCerts(lastNotifiedTime, provider);
+            tryUpdateRecords(lastNotifiedServer, lastNotifiedTime, result);
+        } catch (Exception ex) {
+            LOGGER.error("DynamoDB updateUnrefreshedCertificatesNotificationTimestamp Error: {}/{}", ex.getClass(), ex.getMessage());
+            return false;
+        }
+
+        return true;
     }
 
     @Override
     public List<X509CertRecord> getNotifyUnrefreshedCertificates(String lastNotifiedServer, long lastNotifiedTime) {
-        // Currently unimplemented for AWS
-        return new ArrayList<>();
+
+        Map<String, AttributeValue> expressionAttributeValues = new HashMap<>();
+        expressionAttributeValues.put(":lastNotifiedServerVal", new AttributeValue().withS(lastNotifiedServer));
+        expressionAttributeValues.put(":lastNotifiedTimeVal", new AttributeValue().withN(Long.toString(lastNotifiedTime)));
+
+        ScanRequest scanRequest = new ScanRequest()
```

Comment:
Note that we will need to enable Scan DynamoDB for the IAM role in prod in order for scanning to work.